### DISCUSSION TITLE
Cannot install version 3.9 into Blender 3.4.1

### DISCUSSION THREAD

**Discussion by KenCarter01** [nutti/Screencast-Keys#163] [Q&A]
Not much to help with. Indicates that modules are loaded into AppData\Roaming\Blender Foundation\Blender\3.4\scripts. On verification the files are there. However, Screencast Keys does not come up on the addon menu nor does it come up on the tool bar in Blender.

Any thoughts?

**Comment by nutti** [comment]
@KenCarter01 

Is there any error on the system console?

**Comment by KenCarter01** [comment]
Very strange. I am obviously confused or hit a wrong item. After removing the addon, I redownloaded it, called up blender so that I could try reinstalling the addon at the same time I had this discussion open. God must be laughing at me. This time everything worked. So please ignore this discussion.  Don't know what to say but thankyou.

**Comment by nutti** [comment]
@KenCarter01 

Glad to hear that.
I will close this for now.
